docs/overview.rst

```diff
@@ -255,58 +258,96 @@ information about the cluster. This includes:
 
-* **Version**: This indicates the version number of CrateDB the cluster is
-  running.
+Next Steps
+~~~~~~~~~~
 
-* **Date created**: The day of the original deployment of the cluster.
+* **Connect to your cluster**: Click the "view code snippets" link to
+  see a list of snippets for the available clients and libraries. These
+  include: CLI clients, Java, JavaScript, PHP, Python, and Ruby.
 
-* **DB Username**: This shows once more the username associated with the
-  cluster. You chose this username while setting up the cluster originally.
+* **Demo data**: Clicking the "import the demo data" will show you two
+  snippets that can be used in the SQL Console of your Admin UI. The first
+  snippet a "nyc_taxi" table, and the second one will populate it with our 
+  demo dataset.
 
-* **Plan**: This shows what subscription plan the cluster is running on. For
-  more information on our plans, see the documentation on
-  `subscription plans`_.
+* **Backups**: The "see my backups" will take you to the Backups tab, where 
+  you can see all your backups.
 
-* **Subscription**: The name of the subscription itself, with a link to the
-  :ref:`Subscriptions tab of the Organization page<overview-org-overview>`.
 
 For more information on the CrateDB concepts used here, refer to the `CrateDB
 architecture documentation`_ or the :ref:`glossary<glossary>`.
 
+.. _overview-cluster-overview-access:
 
-.. _overview-cluster-overview-metrics:
+Access
+------
+
+The access tab contains your credentials, security settings, allowlist, and
+information about private endpoints.
 
-Configuration, Usage, and Metrics
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. image:: _assets/img/cluster-access.png
+   :alt: Cloud Console cluster access tab
 
-On the rest of the page you will find various cluster metrics. In the center
-you will see an overview of general usage in the past week, including storage,
-memory, and CPU usage. A bar at the top indicates the 85% mark: if your CPU or
-storage usage is above this percentage of the cluster capacity, you should
-consider upgrading by `scaling the cluster`_.
+.. _overview-cluster-overview-credentials:
 
-On the right you will see more immediate metrics, collected from the last hour
-of cluster operations. This includes average queries, average query response
-time, and disk, memory, and CPU usage for the last hour.
+Credentials
+~~~~~~~~~~~
 
-The queries and query response time of the cluster can be seen in more detail
-on the Cluster Metrics page, for which see below. This page can also be
-accessed by clicking the *View metrics* link at the top right.
+Here you can change your CrateDB Console password.
 
+.. _overview-cluster-overview-security:
+
+Security
+~~~~~~~~
+
+Click the box labelled *Deletion Protected* to add deletion protection to your
+cluster. This will ensure that it can only be deleted in either the CrateDB
+Console or croud by a user with :ref:`org admin rights <org-roles>` (or by a
+Crate.io employee superuser).
```

Comment:
This is not the case - deletion protection ensures that it cannot be deleted by anyone, unless deletion protection is switched off. The intent here is to make deletion a doubly-conscious choice and not accidentally delete your production cluster.